CovidSafe/CovidSafe.DAL/Repositories/Cosmos/CosmosMatchMessageRepository.cs

```diff
@@ -43,6 +42,17 @@ public CosmosMatchMessageRepository(CosmosContext dbContext) : base(dbContext)
             );
         }
 
+        /// <summary>
+        /// Returns the oldest allowed data timestamp for a Cosmos record
+        /// </summary>
+        /// <returns>Oldest allowed timestamp, in ms since UNIX epoch</returns>
+        private long _getMaxDataAge()
```

Comment:
[nit] "Age" seems like not absolute timestamp but (Now() - OldestTimestamp()) (which is 14 days in our case), maybe something like getOldestAllowedTimestamp

Reply:
Similar to your below, I think that makes more sense. I'll leave the configuration parameter name unchanged but will update this function name.